libs/shared/utils/src/BigDecimal.ts

```diff
@@ -0,0 +1,108 @@
+import { formatUnits, parseUnits } from 'viem';
```

Comment:
Nice utility lib

Reply:
I'm still not sure if we should use it... I think this kind of abstraction would root deeply in the code base and might not be sustainable... Anyway, I'll leave it there for now, it's not used anywhere, let's see if we want to keep it or not